src/mlpack/methods/kmeans/kill_empty_clusters.hpp

```diff
@@ -51,13 +51,18 @@ class KillEmptyClusters
       const size_t emptyCluster,
       const arma::mat& /* oldCentroids */,
       arma::mat& newCentroids,
-      arma::Col<size_t>& /* clusterCounts */,
+      arma::Col<size_t>& counts /* clusterCounts */,
```

Comment:
Let's use ``clusterCounts`` instead of ``counts`` here to match with the parameter comment above, the reason for /* clusterCounts */ is basically to avoid a parameter unused warning.